src/lib/mina_base/fee_transfer.ml

```diff
@@ -49,7 +49,7 @@ module Make_str (A : Wire_types.Concrete) = struct
     let fee_token { fee_token; _ } = fee_token
 
     module Gen = struct
-      let with_random_receivers ?(min_fee = 0) ~keys ~max_fee ~token :
+      let with_random_receivers ?(min_fee = 0) ~keys ~max_fee token :
```

Comment:
`keys` should probably be the positional argument, rather than token.

Reply:
Thanks for pointing that out. I made the changes.